api/src/main/java/org/openmrs/module/appointmentscheduling/AppointmentDTO.java

```diff
@@ -0,0 +1,73 @@
+package org.openmrs.module.appointmentscheduling;
+
+public class AppointmentDTO {
```

Comment:
I'd love to come up with a better same for this object, but I'm struggling! 

AppointmentRequest seems like the obvious one, but we are alread using it for something slighlty different.

Let me brainstorm the name for this and AppointmentResource...

Reply:
Let's change this from AppointmentDTO to AppointmentRequisition... then we add should some comments to this class and AppointmentRequest and explain what they are and how they differ...

* AppointmentRequest: persisted object that reflects a request by a provider (or other user) for a scheduler to schedule a patient for a future appointment; has an associated status of pending, fulfilled, or cancelled
* AppointmentRequisition: a transient object used to pass the details of a request for a appointment for immediately fulfilment by the underlying scheduling logic

( Still not in love with that, but hopefully it starts to make it clearer)

Reply:
This makes sense, i'll update the code and let you know